### ISSUE TITLE
PG(121,31) and PG(121,38)

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/primgrp#38]
We have 
```
11^2:(5 x Q_12) : [ [ 121, 31 ], [ 121, 38 ] ]
  [ 121, 31 ] non-isomorphic to [ 121, 38 ] : 
    (C11 x C11) : (C5 x (C3 : C4))
    (C11 x C11) : (C5 x (C3 : Q8))
```
but 
```
gap> G:=PrimitiveGroup(121,31);
11^2:(5 x Q_12)
gap> H:=PrimitiveGroup(121,38);
11^2:(5 x Q_12)
gap> Size(G);
7260
gap> Size(H);
14520
```
so it seems to be a wrong name for `PrimitiveGroup(121,38)`.

**Comment by fingolfin** [comment]
The name should indeed be `11^2:(5 x Q_24)`. Where I assume `Q_{4n}` denotes a *generalized* quaternion group resp. dicyclic group.

This can be verified in various ways, e.g. simply by brute force (factor out the 11^2 and then the 5 or simply brute force all quotients of the "right size"), then check for isomorphism with `DicyclicGroup(24)`:
```
gap> quots:=Filtered(List(NormalSubgroups(G), N -> G/N), q->Size(q)=Size(G)/(11^2*5));
[ Group([ f1, f2, f3 ]) ]
gap> IdGroup(quots[1]);
[ 12, 1 ]
gap> IdGroup(DicyclicGroup(12));
[ 12, 1 ]

gap> quots:=Filtered(List(NormalSubgroups(H), N -> H/N), q->Size(q)=Size(H)/(11^2*5));
[ Group([ f1, f2, f3, f4 ]) ]
gap> IdGroup(quots[1]);
[ 24, 4 ]
gap> IdGroup(DicyclicGroup(24));
[ 24, 4 ]
```

**Comment by alex-konovalov** [comment]
@fingolfin thanks - a  fix now included in PR #36.
